commonv5/ts/sonarqube/TaskReport.ts

```diff
@@ -53,11 +73,8 @@ export default class TaskReport {
       taskReportGlob = path.join("**", REPORT_TASK_NAME);
       taskReportGlobResult = tl.findMatch(tl.getVariable("Agent.BuildDirectory"), taskReportGlob);
     } else {
-      const reportTaskFile = tl.getVariable("SONARQUBE_SCANNER_REPORTTASKFILE");
-      if (!reportTaskFile) {
-        throw TaskReport.throwInvalidReport("[SQ] Unable to find report task file.");
-      }
-      taskReportGlob = reportTaskFile;
+      taskReportGlob =
+        tl.getVariable("SONARQUBE_SCANNER_REPORTTASKFILE") || TaskReport.getDefaultPathGlob();
       taskReportGlobResult = tl.find(taskReportGlob);
```

Comment:
I think you need to use tl.findMatch here instead of find in case we have the ** pattern on it.